uarray/_uarray.pyi

```diff
@@ -13,6 +16,9 @@ from uarray._typing import (
     _SupportsUADomain,
 )
 
+_P = ParamSpec("_P")
+_ReturnT = TypeVar("_ReturnT", bound=tuple[uarray.Dispatchable, ...])
```

Comment:
I would make this `Any`, the output isn't matched.

Reply:
For now I've simply removed the return-typevar. 

It could considered to reintroduce it at some point with its default set to `Any` (so users can more easily manually declare the return-type), but this might be somewhat tricky as the return type can very easily change again by selecting a different backend (via the likes of `uarray.set_backend`). Simply sticking to `Any` for now is probably the safest and easiest option.